vividus-bdd-engine/src/main/java/org/vividus/bdd/steps/ComparisonRule.java

```diff
@@ -94,6 +96,12 @@ public static ComparisonRule parse(String sign)
                 .findFirst();
     }
 
+    @Override
+    public String toString()
+    {
+        return asString;
```

Comment:
```java
return name().toLowerCase().replace('_', ' ');
```
?

Reply:
I considered an extra code execution for each toString() method call, when values already defined, as an overhead